src/transform.h

```diff
@@ -0,0 +1,35 @@
+#ifndef ED_LOCALIZATION_TRANSFORM_H_
```

Comment:
Why do we need this? Don't we have all these things in geolib?

Reply:
It had been here for a long time in an other file. But I removed it.